pytest_django/plugin.py

```diff
@@ -66,9 +66,9 @@ def pytest_addoption(parser):
     group._addoption('--migrations',
                      action='store_false', dest='nomigrations', default=False,
                      help='Enable Django migrations on test setup')
-    group._addoption('--debug-mode',
-                     help='Set DEBUG to true prior to run tests')
+    group._addoption('--django-debug',
                      action='store_true', dest='debug_mode', default=False,
+                     help='Set DEBUG to true prior to run tests')
```

Comment:
nitpick (no need to change it already): s/true/True ?!

Reply:
I'll change it. Thanks.